File: DKS-API/DTOs/PrdRfidAlertDto.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class PrdRfidAlertDto
    {
        public string Gate { get; set; }
        public DateTime Time { get; set; }
        public string Seq { get; set; }
        public string Epc { get; set; }
        public string Reason { get; set; }
        public string Updater { get; set; }
        public string UpdateTime { get; set; }
        
    }
}

File: DKS-SPA/src/app/core/_models/prd-rfid-alert-dto.ts
import { ModelInterface } from "./interface/model-interface";

export class PrdRfidAlertDto implements ModelInterface{
     gate :string;
     area: string;
     time :Date;
     epc  :string;
     seq  :string;
     reason  :string;
     updater  :string;
     updateTime  :string;
     checked:boolean;  //only 4 前端
}
